docs/configuring-metabase/environment-variables.md

```diff
@@ -348,6 +348,15 @@ Default: `null`
 
 The port for [MB_DB_HOST](#mb_db_host).
 
+
+### `MB_DB_QUERY_TIMEOUT_MINUTES`
+
+Type: integer<br>
+Default: `180`
```

Comment:
It says 180 here and 20 in the code, which one is correct?

Reply:
Those docs are auto generated as per [Cam's comment](https://github.com/metabase/metabase/pull/44759#discussion_r1680017204). I've checked output of `metabase.cmd.env-var-dox/generate-dox!` to verify that file has correct content.